package.json

```diff
@@ -19,14 +19,12 @@
     "publish": "rimraf public_html && cp -r dist public_html",
     "serve-static": "ws -d dist/ -s index.html -p 4242",
     "start": "mkdir -p dist && cp res/zee-worker.js dist/ && NODE_ENV=development node server.js",
-    "start-no-hot": "npm run start",
-    "start-prod": "npm run build-prod && npm run serve-static",
-    "start-prod-readable": "npm run build-prod-readable && npm run serve-static",
-    "test": "npm run build && npm run test-all",
-    "test:quiet": "npm run build:quiet && npm run test-all",
-    "test-all": "NODE_ENV=test nyc mocha --compilers js:babel-core/register 'src/test/**/*.js'",
-    "test-file": "NODE_ENV=test mocha --compilers js:babel-core/register",
-    "test-report": "nyc report --reporter=text-lcov > coverage.lcov"
+    "start-no-hot": "npm run start --",
+    "start-prod": "npm run build-prod && npm run serve-static --",
+    "start-prod-readable": "npm run build-prod-readable && npm run serve-static --",
+    "test": "npm run build && npm run test-all --",
```

Comment:
The loaders is one issue, the worker is another issue... This is also in my TODO list indeed.